packages/bundle-size/package.json

```diff
@@ -10,6 +10,7 @@
     "type-check": "tsc"
   },
   "dependencies": {
+    "@azure/data-tables": "^12.0.0-beta.3",
```

Comment:
You'll need to create a custom version group for these dependencies so that syncpack works.. like this: https://github.com/microsoft/fluentui/blob/c5c66cb39e6906f506bf30c9e56c0839f10e2d51/package.json#L387